### PR TITLE
Completely close Exporter when no ground is detected

### PR DESCRIPTION
**Pull request by nicolasespinoza**
### Jira
- Issue 1169

|    DOD    | Done |    Reviewer    | Issue | Notes |
|-----------|------|----------------|-------|-------|
| Code      | ❌   | Liam Wang / Alex Carter / Mat Moradi |  1169|       |
| Unit Test | N/A  | N/A            |       |       |
| Merged    | ❌  |                |       |       |

### CODE REVIEW

File: exporters/InventorRobotExporter/InventorRobotExporter/Exporter/SkeletonBuilder.cs [hunk 1]
```diff
@@ -39,6 +37,10 @@ public static RigidNode_Base ExportSkeleton(IProgress<ProgressUpdate> progress)
                 WinFormsUtils.ShowErrorDialog("Please ground a part in your assembly to export your robot.", "No Ground");
                 return null;
             }
+
+            //Building Model...
```
Comment (wangl5) [inline exporters/InventorRobotExporter/InventorRobotExporter/Exporter/SkeletonBuilder.cs]: Why was this progress report moved?

Comment (nicolasespinoza) [reply exporters/InventorRobotExporter/InventorRobotExporter/Exporter/SkeletonBuilder.cs]: I think I was experimenting with the progress report to see whether I could get rid of the notification, and I left it there. There aren't any noticeable effects from what I can see, but I'll move it back.

File: exporters/InventorRobotExporter/InventorRobotExporter/Exporter/SkeletonBuilder.cs [hunk 2]
```diff
@@ -38,6 +38,7 @@ public static RigidNode_Base ExportSkeleton(IProgress<ProgressUpdate> progress)
             catch (NoGroundException)
             {
                 WinFormsUtils.ShowErrorDialog("Please ground a part in your assembly to export your robot.", "No Ground");
+                RobotExporterAddInServer.Instance.RobotDataManager.hasGround = false;
```
Comment (wangl5) [inline exporters/InventorRobotExporter/InventorRobotExporter/Exporter/SkeletonBuilder.cs]: Could you make this a generic `wasForceQuit` flag and have it enable in the force quit function so this works with other cases where the environment is force quit?

Comment (nicolasespinoza) [reply exporters/InventorRobotExporter/InventorRobotExporter/Exporter/SkeletonBuilder.cs]: Sure

**Pull request merged**